torchao/experimental/ops/linear_8bit_act_xbit_weight/packed_weights_header.h

```diff
@@ -17,19 +17,18 @@ torchao::ops::PackedWeightsHeader get_packed_weights_header_universal(
     int nr,
     int kr,
     int version = 1) {
-  TORCHAO_CHECK(
-      version >= 0 && version < 256, "version must be between 0 and 255");
```

Comment:
What is wrong with this check?

Reply:
Before it existed to make sure version could be packed into a char (since both version/weight_nbit were packed into an unsigned short).  But the header now has more space, and version is now an int, so this bound check is no longer needed.